onboarding/src/Components/Employee/EmployeeFormPages/EmployeeFormPagesList.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const EmployeeFormPagesList = ({ pagesList, switchPage }) => {
+  const moveToForm = (page) => {
```

Comment:
Usually it is better to add `a.preventDefault()` for `<a />` like (*)
So browser will not try to run common event for <a>
;-)